aboutus.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./aboutus.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="about_text">
+        <p class="about_text_title">-Our Mission-</p>
+        <p class="about_p" style="text-align: center;">
+            To provide the most actionable Real Estate Website.
+        </p>
+        <p class="about_text_title">About</p>
+
+        <p class="about_p">
+            As a team, we wanted to make the process of getting your own place easier, so we focused on making the browser simple so that the user would not have to figure how it work first, In addition to providing all the services that the user would need.
+        </p>            
```

Comment:
use prettier correctly to format your code  

![Peek 2021-03-02 19-15](https://user-images.githubusercontent.com/47992412/110006708-a5442a00-7d22-11eb-83ca-4b5ed5846d20.gif)